pootle/apps/pootle_store/models.py

```diff
@@ -211,6 +210,9 @@ def stringcount(string):
         return 1
 
 
+TMServer = SearchBroker()
```

Comment:
@dwaynebailey I had to read this line to know that `SearchBroker` is actually for translation memory. I suggest renaming it to a more descriptive name like `TMBroker`.